pkg/autoscaler/statforwarder/processor.go

```diff
@@ -130,11 +120,17 @@ func (p *remoteProcessor) process(sm asmetrics.StatMessage) error {
 
 	c := p.getConn()
 	if c == nil {
-		c, err = websocket.NewDurableSendingConnectionGuaranteed(p.svcDNS, establishTimeout, p.logger)
+		for _, addr := range p.addrs {
```

Comment:
Comment that we are trying to establish the connection to the pod first directly, then to the svc here and will accept either that works.

Reply:
I think the field comment now covers this.  LMK if you think I should adjust anything.